fix-point/src/Rename.hs

```diff
@@ -1,11 +1,23 @@
 module Rename where 
 
 import Data.Text (Text)
-import Data.Semigroup
+import Data.Semigroup ((<>))
 
 import qualified Data.Text as T
 import qualified Data.Text.IO as TIO
 
+-- Tests to copy and paste into stack repl, make sure in src folder
+{-
+:{
+a = changeLine "CHANGE" "import Lib.App (AppEnv (..)) <- TEST 1"
+b = changeLine "CHANGE" "module Lib where <- TEST 2"
+c = changeLine "CHANGE" "( module Lib ) <- TEST 3"
+d = changeLine "CHANGE" ", module Lib <- TEST 4"
+e = changeLine "CHANGE" "import Servant.Server (serve) <- TEST 5"
```

Comment:
You're calling `changeLine` function once per every line. But what I would to see is passing single `Text` to `changeLine` function, som ideally This comment should look like this:

```haskell
{- Tests to copy and paste into stack repl, make sure in src folder

:{
T.putStrLn $ changeLine "CHANGE" "very long multiline string here"
:}
-}